docs/practitioners/README.md

```diff
@@ -7,6 +7,7 @@ The practitioner resource allows the user to gather information about a medical
 
 | Operation | Description |
 | --------- | ----------- |
-| [Find Practitioner](findPractitioner.md) | Find medical professional allows you to provide a medical professional's personal information in return for their federation ID |
-| [Retrieve Profile](retrieveProfile.md) | Retrieve Profile allows you to provide a medical professional's federation ID in return for their profile information |
-| [Retrieve Board Orders and Licenses](retrieveBOandL.md) | RetrieveBoard Orders and Licenses allows you to provide a medical professional's federation ID in return for their Board Order and License information |
+| [Find Practitioner](findPractitioner.md) | Find medical professional allows you to provide a medical professional's personal information in return for their Federation ID (FID) |
```

Comment:
They would have to dig to find the meaning of FID. Consider moving the definition of FID to the root readme and just use FID everywhere else.
Might want to talk with Cyndi but it might also make sense to clarify this is a 9 digit value because the calls will fail if it is not valid.